packages/suite/src/components/suite/Preloader/index.tsx

```diff
@@ -15,6 +15,7 @@ import Firmware from '@firmware-views';
 import Onboarding from '@onboarding-views';
 import Recovery from '@suite/views/recovery';
 import Backup from '@backup-views';
+import { ModalRoute } from '@suite/constants/suite/routes';
```

Comment:
```suggestion
import { ModalRoute } from '@suite-constants/routes';
```
use aliases pls (and disable auto import plugin for this project)

Reply:
Fixed. Resolving. Thx.